src/helpers/sort-sizes.ts

```diff
@@ -0,0 +1,9 @@
+const sortOrder = ["XS", "S", "M", "L", "XL"];
```

Comment:
We need more sort option
[Size](https://github.com/ZenBit-Tech/sunny-bunnies_fe/blob/develop/src/pages/profile-board/constants.ts/size.ts)